PosteAPI/views.py

```diff
@@ -489,18 +490,28 @@ def patch(self, request, id):
         try:
             post = Post.objects.get(pk=id)
             data = json.loads(request.body.decode('utf-8'))
-            post.edit(data.get('title'), data.get('description'), data.get('url'))
+            tagsMerged = data.get('tags')
+            tag_names = [tag.strip() for tag in tagsMerged.split(', ') if tag.strip()]
+            tag_list = []
+            if tag_names:
+                for tag_name in tag_names:
+                    tag, created = Tag.objects.get_or_create(name=tag_name)
+                    print(f"Tag: {tag}, Created: {created}")
+                    tag_list.append(tag)
+            post.edit(data.get('title'), data.get('description'), data.get('url'), tag_list)
             return Response({"success": True}, status=status.HTTP_200_OK)
         except ObjectDoesNotExist:
```

Comment:
Ideally, should catch "Post.DoesNotExist" and "Post.MultipleObjectsReturned". I am not a fan of a general Exception, though other exceptions *could* occur here and I guess catching it for a general 500 isn't the worst thing in the world. Just for awareness purposes.